lib/OpenQA/Utils.pm

```diff
@@ -224,80 +226,106 @@ sub needle_info {
 }
 
 # logging helpers
-sub log_debug {
-    my ($msg, $channel) = @_;
 
-    if ($channel && $channels{$channel}) {
-        $channels{$channel}->debug($msg);
-    }
-    elsif ($app && $app->log) {
-        $app->log->debug($msg);
-    }
-    else {
-        STDOUT->printflush("[DEBUG] $msg\n");
-    }
+sub log_debug {
```

Comment:
please add some API documentation on how to use it - I'm a bit lost meanwhile, but at least most usages look fine